### ISSUE TITLE
WMA freezes cmus

### ISSUE THREAD

**Issue by amangrewal** [cmus/cmus#895]
cmus freezes when playing wma files. SIGKILL is required to stop cmus (SIGTERM doesn't work). ffplay can play the files correctly.

My best guess is that the issue is with the ffmpeg plugin. I used to be able to play wma without issue, and I recently updated ffmpeg.

Here's the backtrace from gdb
```
#0  0x00007ffff7bb8b48 in __memmove_sse2_unaligned_erms () from /usr/lib/libc.so.6 
#1  0x00007ffff494147b in avpriv_copy_bits () from /usr/lib/libavcodec.so.58   
#2  0x00007ffff4fcad59 in ?? () from /usr/lib/libavcodec.so.58 
#3  0x00007ffff4fcfdbd in ?? () from /usr/lib/libavcodec.so.58   
#4  0x00007ffff49daa60 in ?? () from /usr/lib/libavcodec.so.58 
#5  0x00007ffff49db768 in avcodec_send_packet () from /usr/lib/libavcodec.so.58   
#6  0x00007ffff6823ab5 in ffmpeg_fill_buffer (ic=0x5555558cb4c0, cc=0x5555558961c0, input=0x5555558c8520,  
    output=0x5555558c7090, swr=0x5555559fdac0) at ip/ffmpeg.c:338   
#7  0x00007ffff6823c85 in ffmpeg_read (ip_data=0x555555888538,     
    buffer=0x55555567e6b0 "Q(\035\032a'C\031\220&m\030\342&\214\030\017'{\030J&9\027e%\336\025\023%\254\025]%M\026\064%D\026\005$\357\024\027#v\023\320\"\250\022&\"", count=60480) at ip/ffmpeg.c:394 
#8  0x000055555558112b in ip_read (ip=0x555555888530,
    buffer=0x55555567e6b0 "Q(\035\032a'C\031\220&m\030\342&\214\030\017'{\030J&9\027e%\336\025\023%\254\025]%M\026\064%D\026\005$\357\024\027#v\023\320\"\250\022&\"", count=60480) at input.c:695 
#9  0x000055555558b4d0 in _prebuffer () at player.c:599
#10 0x000055555558cc05 in player_set_file (ti=0x7fffcc006180) at player.c:1171    
#11 0x0000555555574136 in run_command (buf=<optimized out>) at command_mode.c:2883
#12 0x000055555556a7a1 in u_getch () at ui_curses.c:2163   
#13 main_loop () at ui_curses.c:2272test-song-6.mp2 
#14 main (argc=<optimized out>, argv=<optimized out>) at ui_curses.c:2556     
```

1) `uname -a`:
```
Linux hostname 5.0.5-arch1-1-ARCH #1 SMP PREEMPT Wed Mar 27 17:53:10 UTC 2019 x86_64 GNU/Linux
```

2) `cmus --version`:
```
cmus v2.8.0-16-ge429ddf
```

3) `cmus --plugins`:
```
Input Plugins: /usr/lib/cmus/ip
  mad:
    Priority: 55
    File Types: mp3 mp2
    MIME Types: audio/mpeg audio/x-mp3 audio/x-mpeg
  flac:
    Priority: 50
    File Types: flac fla
    MIME Types:
  aac:
    Priority: 50
    File Types: aac
    MIME Types: audio/aac audio/aacp
  wavpack:
    Priority: 50
    File Types: wv
    MIME Types: audio/x-wavpack
  mpc:
    Priority: 50
    File Types: mpc mpp mp+
    MIME Types: audio/x-musepack
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  modplug:
    Priority: 50
    File Types: mod s3m xm it 669 amf ams dbm dmf dsm far mdl med mtm okt ptm stm ult umx mt2 psm
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  opus:
    Priority: 50
    File Types: opus
    MIME Types:
  cdio:
    Priority: 50
    File Types:
    MIME Types: x-content/audio-cdda
  vorbis:
    Priority: 50
    File Types: ogg oga ogx
    MIME Types: application/ogg audio/x-ogg
  mp4:
    Priority: 50
    File Types: mp4 m4a m4b
    MIME Types:
  ffmpeg:
    Priority: 30
    File Types: aa aac ac3 aif aifc aiff ape au fla flac m4a m4b mka mkv mp+ mp2 mp3 mp4 mpc mpp ogg opus shn tak tta wav webm wma wv
    MIME Types:

Output Plugins: /usr/lib/cmus/op
  pulse
  alsa
  oss
  jack
```

**Comment by SiIky** [comment]
I had issues with FLAC and MP3 aswell (some FLAC files played fast and some MP3 files freezed cmus, using up 100% of CPU) :(
BUT! When I looked at the plugins neither FLAC nor MAD were listed. I built cmus again explicitly asking for FLAC and MAD (last time manually changed `config.mk` to have FLAC and MAD support) and now both FLAC and MAD plugins are listed, and the previously problematic files now play fine.
Long story short, I agree that this may be an issue with the FFMPEG plugin.
